plugins/telink-dfu/telink-dfu.quirk

```diff
@@ -1,7 +1,25 @@
 # 8266 mouse beta test
 [BLUETOOTH\VID_248A&PID_8266]
 Plugin = telink_dfu
+GType = FuTelinkDfuBleDevice
+Nickname = Telink BT Mouse Prototype
```

Comment:
What's `Nickname`? Do you mean `Name` or `Summary` perhaps?

Reply:
Should be 'Name'; overwritten during successive commits and fixed in the last commit.